dev-tools/scripts/requirements.txt

```diff
@@ -1,8 +1,8 @@
-six>=1.11.0
-Jinja2>=2.10.1
-PyYAML>=5.1
-holidays>=0.9.10
-ics>=0.4
-console-menu==0.5.1
-PyGithub
-jira
\ No newline at end of file
+six==1.16.0
+Jinja2==3.1.1
+PyYAML==6.0
+holidays==0.16
+ics==0.7.2
+console-menu==0.7.1
+PyGithub==1.56
+jira==3.4.1
```

Comment:
I pinned every dependency here. 
I releaize that these are shared between release wizard and the other py tools in this folder, so I cannot guarantee that all the other tools work with the pinned version. I tested `githubPRs.py` and it works.

Reply:
Changed these to [`~=` Compatible Release](https://peps.python.org/pep-0440/#compatible-release) to allow bugfix upgrades.